src/pages/guidelines/pictograms/usage.mdx

```diff
@@ -7,104 +7,172 @@ tabs: ['Library', 'Usage', 'Code', 'Contribute']
 ---
 
 import { Bee32 } from '@carbon/icons-react';
+import { noGutterSm } from '../../../styles/Grid.module.scss';
```

Comment:
@peter-garvin this relative path might be wrong?

Reply:
The Vercel errors are calling out incorrect paths to `Grid.module.scss`.

![image](https://user-images.githubusercontent.com/1691245/113048097-c2281d80-9167-11eb-86c7-d07d3aef9f0f.png)

Reply:
I don't know that IDL website code base, but that line was attempting to use a generated class name from a file that only lives in the IDL website. The build should pass by removing that line, unless there were other issues as well.